config.sh

```diff
@@ -1,7 +1,9 @@
+
+EXCLUDE_IP=${EXCLUDE_IP:=$(hostname -i)}
```

Comment:
that doesn't make any sense. Even if the user wants to exclude one of his servers, why would he want that to cause the relay to start sending its own logs..??